src/libs/humanizer/index.ts

```diff
@@ -44,6 +49,11 @@ export const humanizerCallModules: HumanizerCallModule[] = [
   genericErc721Humanizer,
   gasTankModule,
   uniswapHumanizer,
+  curveModule,
+  traderJoeModule,
+  KyberSwap,
+  AcrossModule,
+  OneInchModule,
```

Comment:
Not related but the displayed label for approvals is wrong. 
![image](https://github.com/AmbireTech/ambire-common/assets/68795596/bf64e612-cce1-4936-84ae-f4553b1416b3)